java/README.md

```diff
@@ -20,14 +20,14 @@ In the `src/main` directory of the project, you will find several sub-directorie
 3. Run `mvn clean install` from the `java` directory to build the project and generate required sources from the proto file.
 4. The `arguments.yaml` file in the `src/main/resources` sub-directory contains a list of required and optional configurations needed to run the examples. The file contains detailed comments on how to set the configurations.
 5. Get the username, password, and login URL of the Salesforce org you wish to use.
-6. For the examples in `genericpubsub` package, a custom **_CarMaintenance_** [platform event](https://developer.salesforce.com/docs/atlas.en-us.platform_events.meta/platform_events/platform_events_define_ui.htm) has to be created in the Salesforce org. Ensure your CarMaintenance platform event matches the following structure:
+6. For the examples in `genericpubsub` package, a custom **_Order_event_** [platform event](https://developer.salesforce.com/docs/atlas.en-us.platform_events.meta/platform_events/platform_events_define_ui.htm) has to be created in the Salesforce org. Ensure your Order_Event platform event matches the following structure:
    - Standard Fields
-       - Label: `CarMaintenance`
-       - Plural Label: `CarMaintenances`
+       - Label: `Order_Event`
+       - Plural Label: `Order_Events`
    - Custom Fields
-       - `Cost` (Number)
-       - `Mileage` (Number)
-       - `WorkDescription` (Text, 200)
+       - `Order Number` (Text, 18)
+       - `City` (Text, 200)
```

Comment:
Should the City and Order Number match what is in the platform events doc in https://developer.salesforce.com/docs/atlas.en-us.platform_events.meta/platform_events/platform_events_filter_pe.htm?

City is Text(50)
Amount is Number(16,2)